crypto-jni/Cargo.toml

```diff
@@ -7,9 +7,9 @@ edition = "2018"
 
 [dependencies]
 jni = "0.19.0"
-rand = "=0.7"
-ed25519-dalek = "=1.0"
-curve25519-dalek = "3.0"
+rand = "=0.8"
+ed25519-dalek = "=2.0"
+curve25519-dalek = "4.1"
```

Comment:
I'm surprised the curve is needed as a direct dependency. I can't seem to find where it is used.

Reply:
I just updated it as it was already there. I have just removed it now, so you are right in that it was not in use.